document/core/util/math.def

```diff
@@ -710,6 +710,7 @@
 .. |iles| mathdef:: \xref{exec/numerics}{op-ile_s}{\F{ile\_s}}
 .. |igeu| mathdef:: \xref{exec/numerics}{op-ige_u}{\F{ige\_u}}
 .. |iges| mathdef:: \xref{exec/numerics}{op-ige_s}{\F{ige\_s}}
+.. |iextends| mathdef:: \xref{exec/numerics}{op-iextend_s}{\F{iextend{M}\_s}}
```

Comment:
Minor issue: to get the proper type setting this needs to be defined as {\F[iextend}M\F{\_s}}.
And for consistency, can we name the macro iextendns and the label op-iextendn_s?.